src/server/api/boards/boards.js

```diff
@@ -19,10 +19,19 @@ function createFindQuery(username) {
 }
 
 function idQuery(username, id) {
-  return {
-    'username': username,
-    '_id': id,
-  };
+  try {
+    let mongoId = new mongodb.ObjectId(id);
+
+    return {
+      'username': username,
+      '_id': mongoId,
+    };
+  } catch (error) {
+    return {
+      'username': username,
+      '_id': '',
+    };
+  }
```

Comment:
i'm not convinced that this is a good solution. maybe it should return `null` or the error should be handled where you call the function